xml/xen2kvm.xml

```diff
@@ -791,7 +792,7 @@ getty@xvc1.service -&gt; /usr/lib/systemd/system/getty@xvc1.service
     <title>Export the &xen; &vmguest; configuration</title>
     <para>
      First export the configuration of the guest and save it to a file. A
-     typical one may look like this:
+     typical one might look like this:
```

Comment:
proposal to rephrase to something like: "See the following example file." or "First export [...] to a file. For example: "